### ISSUE TITLE
Mobile - scrolling / editing with inputStyle contenteditable jumps around

### ISSUE THREAD

**Issue by elsigh** [codemirror/CodeMirror#6308]
Here's a gif from one of the demo pages.

![Screen Recording 2020-06-09 at 7 33 30 AM](https://user-images.githubusercontent.com/517210/84162012-c2aec680-aa24-11ea-970b-83ab9efdae98.gif)

I've tried calling `preventDefault()` in a `scrollCursorIntoView` handler. 

When I set `inputStyle: textarea` the scrolling/focus behavior works as expected from a user experience point of view. However with `inputStyle: textarea` it seems we cannot longpress and select text. So kind of stuck between a rock and a hard place =)

**Comment by CWKevo** [comment]
Can confirm that this happens too, using Chrome for Android.

**Comment by rogdevil** [comment]
Hey, I have been assigned on gitpay to resolve this. Can you please assign it to me

**Comment by rogdevil** [comment]
also I need a proper description on how to reproduce the issue it would be a great help

**Comment by elsigh** [comment]
@rogdevil - you can repro like so: 
1) Grab an Android phone
2) Visit any of the demos, but go for https://codemirror.net/demo/html5complete.html
3) Tap into the code's container and then scroll the page down and try to tap into a part of the code and notice that you can't get your tap to put the cursor where you're tapping - sometimes it jumps on me, sometimes it puts the cursor into completely the wrong place, sometimes it scrolls the whole page somewhere else. It's just really hard / frustrating to get your cursor to a specifically tapped spot.

**Comment by rogdevil** [comment]
yes, it does this sometimes. do this same happens on the code in the repo?

**Comment by elsigh** [comment]
@rogdevil I don't understand your question, but you can reproduce this problem with this repository

**Comment by rogdevil** [comment]
yeah sure we can make, I was dumb I didn't saw that ... 🤔

**Comment by WhatMeansGithub** [comment]
This is still an issue. I have this issue on iOS when trying to tap or type or paste anything. It won't type where I want to but jumps somewhere randomly (usually top or middle of a command).

**Comment by dragonDScript** [comment]
Hello there! Can I work on this issue? I have experience with CodeMirror.
